src/editor-page.jsx

```diff
@@ -160,12 +187,19 @@ var EditorPage = React.createClass({
     handleChange: function(toChange, cb, silent) {
         var newProps = _(this.props).pick("question", "hints", "answerArea");
         _(newProps).extend(toChange);
-        this.props.onChange(newProps, cb, silent);
+        this.props.onChange(newProps, () => {
+            if (typeof cb === "function") {
+                cb();
+            }
+            // update the search results after question, hints, or answerArea
+            // changes
+            this.refs.searchAndReplace.updateSearchResults(newProps);
+        }, silent);
```

Comment:
I decided to handle updates to questions, hints, etc. while the search panel is open.  This should make things easier both for both users (because they won't have to close the search panel to make changes) and developers (because we won't have to disable editing while the search panel is open).